app/views/alchemy/admin/pages/fold.js.erb

```diff
@@ -1,3 +1 @@
-$('#page_<%= @page.id -%>').replaceWith('<%= j render("page", page: @page) %>');
-Alchemy.watchForDialogs('#page_<%= @page.id %>');
-Alchemy.Spinner.watch('#page_<%= @page.id %>');
+Alchemy.Sitemap.fetch();
```

Comment:
Ok, I will see. It might be confusing to users to see the whole tree rerendered, after collapsing a sub tree, but it's great enough for now.